build.gradle.kts

```diff
@@ -65,12 +65,8 @@ tasks {
       html.required.set(true) // observe findings in your browser with structure and code snippets
     }
   }
-  named("check").configure {
-    this.setDependsOn(
-      this.dependsOn.filterNot {
-        it is TaskProvider<*> && it.name == "detekt"
-      },
-    )
+  getByName("check") {
+    dependsOn(":ktlintCheck", "detekt")
```

Comment:
there were quite a few issues as detekt wasn't being run and not enforced by the build. This enables it as part of `check` which is run by circle